src/Components/v2/ArtworkFilter/Utils/isDefaultFilter.tsx

```diff
@@ -0,0 +1,19 @@
+export const isDefaultFilter: (name: string, value: any) => boolean = (
```

Comment:
I wonder if we can make this helper go away, based on the way you simplified the reducer `UNSET` action? It's accomplishing something very similar - trying to identify if something is set to its initial value or something different.